evmbin/src/display/json.rs

```diff
@@ -59,25 +95,37 @@ impl Informant {
 	fn informant_trace(informant: &Informant, gas_used: U256) -> String {
 		let info = ::evm::Instruction::from_u8(informant.instruction).map(|i| i.info());
 
-		json!({
-			"pc": informant.pc,
-			"op": informant.instruction,
-			"opName": info.map(|i| i.name).unwrap_or(""),
-			"gas": format!("{:#x}", gas_used.saturating_add(informant.gas_cost)),
-			"gasCost": format!("{:#x}", informant.gas_cost),
-			"memory": format!("0x{}", informant.memory.to_hex()),
-			"stack": informant.stack,
-			"storage": informant.storage,
-			"depth": informant.depth,
-		}).to_string()
+		// Reference: https://serde.rs/attr-skip-serializing.html
+		let trace_data =
+			TraceData {
+				pc: informant.pc,
+				op: informant.instruction,
+				op_name: info.map(|i| i.name).unwrap_or(""),
+				gas: &format!("{:#x}", gas_used.saturating_add(informant.gas_cost)),
+				gas_cost: &format!("{:#x}", informant.gas_cost),
+				memory: &format!("0x{}", informant.memory.to_hex()),
+				stack: &informant.stack,
+				storage: &informant.storage,
+				depth: informant.depth,
+			}
+		;
+
+		serde_json::to_string(&trace_data).unwrap()
```

Comment:
Calling `unwrap` is discouraged according to our [style guide](https://wiki.parity.io/Substrate-Style-Guide).
```suggestion
		serde_json::to_string(&trace_data).expect("serialization cannot fail; qed")
```